engine/access/rest/request/signatures.go

```diff
@@ -0,0 +1,100 @@
+package request
+
+import (
+	"fmt"
+
+	"github.com/onflow/flow-go/engine/access/rest/models"
+	"github.com/onflow/flow-go/engine/access/rest/util"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+type TransactionSignature flow.TransactionSignature
+
+func (s *TransactionSignature) Parse(
+	rawAddress string,
+	rawSignerIndex string,
+	rawKeyIndex string,
+	rawSignature string,
+) error {
+	var address Address
+	err := address.Parse(rawAddress)
+	if err != nil {
+		return err
+	}
+
+	sigIndex, err := util.ToUint64(rawSignerIndex)
+	if err != nil {
+		return fmt.Errorf("invalid signer index: %v", err)
```

Comment:
same for other errors in this file
```suggestion
		return fmt.Errorf("invalid signer index: %w", err)
```

Reply:
fixed in all places